dist/src/main/dist/conf/client-hazelcast.xml

```diff
@@ -14,6 +14,22 @@
         <cluster-members>
             <!--MEMBERS-->
         </cluster-members>
+
+        <ssl enabled="true">
```

Comment:
-->> Are you sure this isn't going to cause problems when we run normal HZ? Because now all the environments want to run with SSL, but this functionality isn't available when we test normal HZ. Even it it was, I don't want this to be the default.